<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>319</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正俊黑简体_中</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget{
	background-color: rgb(50,50,50);
}
QWidget{
	color: rgb(235, 235, 235);
}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>131</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正俊黑简体_中</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
	background-color: rgb(80, 80, 80);
}
QListWidget::item:disabled{
	color:grey;
}

</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>331</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正俊黑简体_中</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit#textEdit{
	background-color: rgb(80, 80, 80);
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_num">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color: rgb(80, 80, 80);
}
QSpinBox::up-button,QSpinBox::down-button{
	width:0px;
}</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正俊黑简体_中</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(223, 223, 223);</string>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_run">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 12pt &quot;Consolas&quot;;
    color:rgb(200,200,200);
	border: 0px solid transparent;
    background-color: rgba(35,35,35,150);
}
QPushButton:disabled{
	color:grey;
}
QPushButton:hover{
	background-color: rgba(137, 144, 143,100);
}

QPushButton:pressed{
	background-color: rgba(99, 99, 99,100);
}</string>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
    background-color:rgb(80,80,80);
    color:white;
    font-family:Consolas;
    border: 1px solid rgb(90,90,90);
}
QComboBox:disabled{
	color:grey;
}
QComboBox::drop-down {
    border: 1px solid rgb(100,100,100);
    width: 5px;
}
QAbstractItemView {
    background-color:rgb(80,80,80);
    color:white;
}
QComboBox::down-arrow{
    width:0px;/*用于隐藏箭头*/
}</string>
   </property>
   <item>
    <property name="text">
     <string>ALL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_shuffle">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>291</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 12pt &quot;Consolas&quot;;
    color:rgb(200,200,200);
	border: 0px solid transparent;
    background-color: rgba(35,35,35,150);
}
QPushButton:disabled{
	color:grey;
}
QPushButton:hover{
	background-color: rgba(137, 144, 143,100);
}

QPushButton:pressed{
	background-color: rgba(99, 99, 99,100);
}</string>
   </property>
   <property name="text">
    <string>Rand</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_noback">
   <property name="geometry">
    <rect>
     <x>363</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正俊黑简体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator:unchecked
{
   background-color:rgb(80,80,80);
}
QCheckBox:disabled{
	color:grey;
}
QCheckBox::indicator:checked
{
	background-color: rgb(225, 225, 225);
}</string>
   </property>
   <property name="text">
    <string>不放回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 12pt &quot;Consolas&quot;;
    color:rgb(200,200,200);
	border: 0px solid transparent;
    background-color: rgba(35,35,35,150);
}
QPushButton:disabled{
	color:grey;
}
QPushButton:hover{
	background-color: rgba(137, 144, 143,100);
}

QPushButton:pressed{
	background-color: rgba(99, 99, 99,100);
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_min">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color: rgb(80, 80, 80);
}
QSpinBox::up-button,QSpinBox::down-button{
	width:0px;
}</string>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="prefix">
    <string>Min:</string>
   </property>
   <property name="minimum">
    <number>-99999999</number>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_max">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color: rgb(80, 80, 80);
}
QSpinBox::up-button,QSpinBox::down-button{
	width:0px;
}</string>
   </property>
   <property name="prefix">
    <string>Max:</string>
   </property>
   <property name="minimum">
    <number>-99999999</number>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_mode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正俊黑简体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator:unchecked
{
   background-color:rgb(80,80,80);
}
QCheckBox::indicator:checked
{
	background-color: rgb(225, 225, 225);
}</string>
   </property>
   <property name="text">
    <string>Num | Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
